src/libcrun/utils.c

```diff
@@ -897,8 +897,23 @@ libcrun_is_apparmor_enabled (libcrun_error_t *err)
   return apparmor_enabled;
 }
 
+static int 
+is_current_process_confined(libcrun_error_t *err)
+{
+  int fd = open("/proc/thread-self/attr/current", O_RDONLY|O_CLOEXEC);
```

Comment:
if you use cleanup_close you don't need to `close()` the fd later:

```
cleanup_close int fd = open ("/proc/thread-self/attr/apparmor/current", O_RDONLY|O_CLOEXEC);
```

Reply:
good to know, thanks.